### DISCUSSION TITLE
METTRE A JOUR PRESTASHOP

### DISCUSSION THREAD

**Discussion by Selleriegz** [PrestaShop/PrestaShop#36785] [Q&A]
Bonjour, je souhaite mettre à jour prestashop avec le module ClickUpgrade, comment puis-je effectuer une  sauvegarde manuelle complète de mes fichiers et de  ma base de donnée ?

**Comment by Quetzacoalt91** [comment]
Shop backup is part of the upgrade process in this module, keep the related options enabled (including images if needed).

You can find additional methods to backup on this page https://devdocs.prestashop-project.org/8/basics/keeping-up-to-date/backup/
